.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: required
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install texlive texlive-latex3
+script:
+- ./build.sh
+deploy:
+  provider: script
+  script: ./travis_deploy.sh
+  skip_cleanup: true
+  on:
+    branch: master
```

Comment:
If you merge this into `frontier`, the branch for the script should also use that branch.

Reply:
Really?  I don't want to deploy PDFs from `frontier`.